### DISCUSSION TITLE
不知如何下載csv檔案

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS3073-C/Python-Community-GS3073-C#60] [Q&A] [answered]
想在github上下載別人公開的csv檔案
上網查是說在Raw點選「另存連結為...」就可以下載
但是我按了之後只能跑出txt檔案的選項，不知道有沒有其他方法能順利下載?

### **這是顯示的檔名**
![csv檔案](https://user-images.githubusercontent.com/90883526/137410693-5b4146c0-36f0-4951-a786-a166324e3582.png)

### **這是我能選的選項**
![txt畫面](https://user-images.githubusercontent.com/90883526/137410698-d10d08f7-9247-4472-8aa3-1ae8a45dfc3f.png)

**Comment by jacky0115** [comment]
aw是???

**Reply by bonny123865** [reply]
抱歉抱歉，是Raw!
![Raw](https://user-images.githubusercontent.com/90883526/137425149-bde39302-6df0-46f4-b19e-ace4eb1c83eb.png)

**Comment by jacky0115** [comment, marked as answer]
你要先點進去才在裡面右鍵另存新檔喔

**Reply by bonny123865** [reply]
完成了，謝謝！
